src/event_message/parse.rs

```diff
@@ -1,20 +1,5 @@
-use super::{
-    AttachedSignaturePrefix,
-    EventMessage,
-    SignedEventMessage,
-    SignedNontransferableReceipt,
-    SignedTransferableReceipt,
-    payload_size::PayloadType,
-};
-use crate::{
-    derivation::attached_signature_code::b64_to_num,
-    event::{event_data::EventData, sections::seal::EventSeal},
-    prefix::{
-        parse::{attached_signature, basic_prefix, event_seal, self_signing_prefix},
-        BasicPrefix, SelfSigningPrefix,
-    },
-    state::IdentifierState,
-};
+use super::{AttachedSignaturePrefix, EventMessage, SignedEventMessage, attachement::{SorceSeal, Attachement}, payload_size::PayloadType, signed_event_message::{SignedNontransferableReceipt, SignedTransferableReceipt}};
```

Comment:
can we agree on import formatting?
Single line vs one import per line - it's kinda mess in every module.
Auto add via IDE will flatten everything into single line, so I guess we can go with that?

Reply:
This can be solved by using `cargo fmt`. Should I reformat it here or it's better to have separate PR for that?